FirebaseVertexAI/Sample/GenerativeAITextSample/Screens/SummarizeScreen.swift

```diff
@@ -28,19 +28,22 @@ struct SummarizeScreen: View {
 
   var body: some View {
     VStack {
-      Text("Enter some text, then tap on _Go_ to summarize it.")
-      HStack(alignment: .top) {
-        TextField("Enter text summarize", text: $userInput, axis: .vertical)
-          .textFieldStyle(.roundedBorder)
-          .onSubmit {
+      VStack(alignment: .leading) {
+        Text("Enter some text, then tap on _Go_ to summarize it.")
+          .padding(.horizontal, 6)
+        HStack(alignment: .top) {
+          TextField("Enter text summarize", text: $userInput, axis: .vertical)
```

Comment:
I think it might be better to manage the [focus](https://developer.apple.com/documentation/swiftui/focusstate) within the view instead of explicitly closing the keyboard. This view actually already has a property `focusedField` for this but it was never hooked up. Adding `.focused($focusedField, equals: .message)` to the `TextField` should automatically modify the state when the user taps in the field.

Reply:
Yes, you are right. Adding now.